key-vault.tf

```diff
@@ -11,6 +11,28 @@ common_tags                = var.common_tags
 create_managed_identity    = true
 }
 
+
+data "azurerm_key_vault" "key_vault" {
+  name                = module.disposer-vault.key_vault_name
+  resource_group_name = azurerm_resource_group.rg.name
+}
+
+data "azurerm_key_vault" "s2s_vault" {
+  name                = "s2s-${var.env}"
+  resource_group_name = "rpe-service-auth-provider-${var.env}"
+}
+
+data "azurerm_key_vault_secret" "key_from_vault" {
```

Comment:
shouldn't this be "key_vault" rather than "key_from_vault"?

Reply:
This is the key we are pulling from s2s vault, _key_vault_  is already defined and used to access our vault.

Reply:
I am fine with that, but `key_vault` defined on `azurerm_key_vault` resource, not on `azurerm_key_vault_secret` resource.